### ISSUE TITLE
Demo link not working

### ISSUE THREAD

**Issue by vanigupta20024** [danialfarid/ng-file-upload#2147]
Demo link in the readme is not working.

**Comment by danialfarid** [comment]
Yes since google app engine doesn't have free tier now and it doesn't let you set budget limit the cost of keeping up the demo page was going up so had to shut it down.

**Comment by vanigupta20024** [comment]
Should I remove it and create a PR? Btw thanks for the prompt response!
